azure-pipelines.yml

```diff
@@ -31,11 +31,18 @@ steps:
     versionSpec: 
 - pwsh: |
     .\build.ps1
+  errorActionPreference: 'continue'
+  ignoreLASTEXITCODE: true
```

Comment:
I don't think we should continue if the build failed. This will give incorrect impression for CI results.

Reply:
have removed this and have updated the next task to run even if this one failed